### PR TITLE
Fix Travis integration

### PR DESCRIPTION
**Pull request by federicoaldunate**
- Fix Travis CI integration
- Add `Build Status` of Travis CI to `README.md`
- Change capybara tests with headless chrome

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 AllCops:
   Exclude:
     - spec/dummy/db/schema.rb
+    - vendor/bundle/**/*
```
Comment (santib) [inline .rubocop.yml]: Why is this?

File: .travis.yml [hunk 2]
```diff
@@ -2,3 +2,19 @@ language: ruby
...
 rvm:
   - 2.5.1
+
+dist: trusty
+sudo: required
+addons:
+  chrome: stable
+
+before_install:
+  - wget https://chromedriver.storage.googleapis.com/2.35/chromedriver_linux64.zip
```
Comment (santib) [inline .travis.yml]: why is this? https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-the-chrome-addon-in-the-headless-mode it doesn't say you need to download chrome

File: .travis.yml [hunk 3]
```diff
@@ -2,3 +2,19 @@ language: ruby
...
+  - unzip chromedriver_linux64.zip -d spec/
+
+before_script:
+  - "export PATH=$PATH:`pwd`/spec/"
+
+script:
```
Comment (santib) [inline .travis.yml]: I don't think you need this. By default it runs `bundle exec rake` which I already configured to run rubocop and rspec in `Rakefile`

File: .travis.yml [hunk 4]
```diff
@@ -2,3 +2,19 @@ language: ruby
 cache: bundler
 rvm:
   - 2.5.1
+
+dist: trusty
+sudo: required
```
Comment (santib) [inline .travis.yml]: why?

File: lib/active_admin_chat.rb [hunk 5]
```diff
@@ -33,4 +33,6 @@ def setup
 
 ::ActiveAdmin.send :include, ActiveAdminChat::ActiveAdmin
 ::ActiveAdmin::Application.send :include, ActiveAdminChat::ActiveAdmin::Application
-::ActiveAdmin::Views::IndexAsTable::IndexTableFor.send :prepend, ActiveAdminChat::ActiveAdmin::Views::IndexAsTable::IndexTableFor
+::ActiveAdmin::Views::IndexAsTable::IndexTableFor.send :prepend,
```
Comment (santib) [inline lib/active_admin_chat.rb]: Let's try to find another way.. maybe:

```
::ActiveAdmin::Views::IndexAsTable::IndexTableFor.send(
  :prepend,
  ActiveAdminChat::ActiveAdmin::Views::IndexAsTable::IndexTableFor
)
```

or

```
::ActiveAdmin::Views::IndexAsTable::IndexTableFor
  .send . :prepend, ActiveAdminChat::ActiveAdmin::Views::IndexAsTable::IndexTableFor
```

File: lib/active_admin_chat/active_admin/application.rb [hunk 6]
```diff
@@ -25,7 +25,8 @@ def show
             def create
               conversation = ActiveAdminChat.conversation_klass
                                             .find_or_create_by!(
-                                              "#{user_relation_name_id}": params[:"#{user_relation_name_id}"]
+                                              "#{user_relation_name_id}":
+                                              params[user_relation_name_id.to_sym]
```
Comment (santib) [inline lib/active_admin_chat/active_admin/application.rb]: another option would be

```
conversation = 
  ActiveAdminChat.conversation_klass
                               .find_or_create_by!(
                                 "#{user_relation_name_id}": params[:"#{user_relation_name_id}"]
                               )
```

I think it's easier to understand this way

File: .rubocop.yml (hunk 1 above)
Comment (federicoaldunate) [reply .rubocop.yml]: I did it, because of this issue https://github.com/rubocop-hq/rubocop/issues/6398

**Pull request merged**